bridge/cypress/support/pageobjects/DashboardPage.ts

```diff
@@ -39,6 +40,15 @@ class DashboardPage {
     cy.get('.dt-button-primary > span.dt-button-label').contains('Create a new project').click();
     return new NewProjectCreatePage();
   }
+
+  public clickStageTag(stage: string): EnvironmentPage {
```

Comment:
This one may be flaky. We only have the stage here and a stage name may be in several projects.
I would introduce a new `uitestid` here for the `ktb-project-tile`. `ktb-project-${projectName}` or something.
Then we could write something like
```ts
cy.byTestId(`ktb-project-${projectName}`).find('dt-tag a').contains(stageName).click()
```